sdf/base_settings.py

```diff
@@ -72,11 +72,13 @@
 
 WSGI_APPLICATION = 'sdf.wsgi.application'
 
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
 EMAIL_USE_TLS = True
-EMAIL_HOST = 'mail.gmx.net'
```

Comment:
Mmh, da müsste ich erst nachschaun wie das geht. Im Prinzip vom develop eine Datei kopieren und einfach in den Branch überschreiben lassen? Eigentlich sind das nur 2 Zeilen.....